packages/katello/katello/hostname-change.rb

```diff
@@ -143,8 +143,7 @@ def next_steps_message
   You will have to install the new bootstrap rpm and reregister all clients and #{@plural_proxy} with subscription-manager
```

Comment:
Please fix this comment.
Maybe something like:

You will have to apply the new katello-ca-consumer and reregister all clients and #{@plural_proxy} with subscription-manager
Comment

Reply:
This seems entirely wrong. Just because the hostname changes does not require re-registration of a host.

Reply:
If we change Foreman's host name, we will definitely need to reconfigure rhsm to point to the new server, and sometimes (depends if the cert is self-managed for example) we will need the CA to be updated. I think that simplifying the procedure takes precedence here, and having a single procedure will do us good.

Reply:
None of that requires a host to be re-registered.